project-base/storefront/components/Layout/Header/HamburgerMenu/HamburgerMenu.tsx

```diff
@@ -4,15 +4,15 @@ import { MouseEventHandler } from 'react';
 import { twJoin } from 'tailwind-merge';
 
 type HamburgerMenuProps = {
-    onClick: MouseEventHandler<HTMLDivElement>;
+    onClick?: MouseEventHandler<HTMLDivElement>;
```

Comment:
I know another way to solve this placeholder would be creating another placeholder component which we already have a lot and we don't want another one, but this is not good in my eyes. Even though it probably won't be problem now we have component which require onClick callback in order to be working but we allow not not send this callback for the sake of the placeholder... 😏

But I think I have sort of solution. Let's require this callback but put some dummy function for the placeholder...?

Reply:
I see your point. Added.